[31m[1mERROR   [0m test_datepicker:test_datepicker.py:117 Test failed with error: Message: move target out of bounds
  (Session info: chrome=137.0.7151.105)
Stacktrace:
0   chromedriver                        0x00000001045e9d14 cxxbridge1$str$ptr + 2735276
1   chromedriver                        0x00000001045e1f88 cxxbridge1$str$ptr + 2703136
2   chromedriver                        0x0000000104132564 cxxbridge1$string$len + 90028
3   chromedriver                        0x00000001041c2744 cxxbridge1$string$len + 680332
4   chromedriver                        0x00000001041bb0c8 cxxbridge1$string$len + 650000
5   chromedriver                        0x000000010416dbe8 cxxbridge1$string$len + 333360
6   chromedriver                        0x00000001045ad598 cxxbridge1$str$ptr + 2487600
7   chromedriver                        0x00000001045b0830 cxxbridge1$str$ptr + 2500552
8   chromedriver                        0x000000010458dc14 cxxbridge1$str$ptr + 2358188
9   chromedriver                        0x00000001045b10b8 cxxbridge1$str$ptr + 2502736
10  chromedriver                        0x000000010457edec cxxbridge1$str$ptr + 2297220
11  chromedriver                        0x00000001045d1420 cxxbridge1$str$ptr + 2634680
12  chromedriver                        0x00000001045d15ac cxxbridge1$str$ptr + 2635076
13  chromedriver                        0x00000001045e1bd4 cxxbridge1$str$ptr + 2702188
14  libsystem_pthread.dylib             0x00000001959adc0c _pthread_start + 136
15  libsystem_pthread.dylib             0x00000001959a8b80 thread_start + 8